### PR TITLE
Removed staging net feature.

### PR DESCRIPTION
**Pull request by EmilLa1**
## Purpose

To remove the staging_net feature as per https://github.com/Concordium/concordium-node/issues/71

## Changes

Removed the staging_net username from the protobuf. 

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: concordium_p2p_rpc.proto [hunk 1]
```diff
@@ -80,7 +80,6 @@ message NodeInfoResponse {
   IsInBakingCommittee consensus_baker_committee = 7;
   bool consensus_finalizer_committee = 8;
   google.protobuf.UInt64Value consensus_baker_id = 9;
-  google.protobuf.StringValue staging_net_username = 50;
```
Comment (abizjak) [inline concordium_p2p_rpc.proto:83]: I don't think we should change the API. We should mark the field as deprecated.

Comment (abizjak) [reply concordium_p2p_rpc.proto:83]: https://developers.google.com/protocol-buffers/docs/proto3

**Review by abizjak — CHANGES_REQUESTED** [review]

File: concordium_p2p_rpc.proto (hunk 1 above)
Comment (abizjak) [reply concordium_p2p_rpc.proto:83]: Or alternatively make it reserved https://developers.google.com/protocol-buffers/docs/proto3#reserved 

But I think the most painless transition will be to mark it deprecated.

Comment (EmilLa1) [reply concordium_p2p_rpc.proto:83]: I suppose you're right if it's "already being used". I'll mark the field as deprecated.

**Review by abizjak — APPROVED** [review]

**Pull request merged**